src/04-arrays.js

```diff
@@ -106,7 +123,9 @@ function removeElementFromEndOfArray(array) {}
  *  addElementToFrontOfArray([ 10 ], 9);
  *  //> [ 9, 10 ]
  */
-function addElementToFrontOfArray(array, element) {}
+function addElementToFrontOfArray(array, element) {
```

Comment:
Let's look at this again.  Remember that ` unshift() ` does add an elemnent to an array but it actually returns the new length of the array and ` toString() ` converts the data to a string.  How can we add to the array using what we have here to return an array with the new element?